contracts/lib/arbitration/ITransactionFeesOracle.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.4.24;
+
+import "../token/ERC20.sol";
+
+
+interface ITransactionFeesOracle {
+    function setFee(bytes32 appId, ERC20 token, uint256 amount) external;
+    function setFees(bytes32[] _appIds, ERC20[] _tokens, uint256[] _amounts) external;
+    function unsetFee(bytes32 _appId) external;
+    function unsetFees(bytes32[] _appIds) external;
+    function getFee(bytes32 appId) external view returns (ERC20, uint256, address);
+}
```

Comment:
Do we need this interface here?

Reply:
You are right!

Reply:
We need  it for `Agreement`. Would you rather put it there?